packages/manager/modules/iam/src/dashboard/applications/applications.module.js

```diff
@@ -0,0 +1,15 @@
+import angular from 'angular';
+
+import component from './applications.component';
+import routing from './applications.routing';
+import deleteModule from './delete';
+
+const moduleName = 'ovhManagerIAMDashboardApplications';
+
+angular
+  .module(moduleName, [deleteModule])
+  .component('iamApplications', component)
+  .config(routing)
+  .run(/* @ngTranslationsInject:json ./translations */);
```

Comment:
No translation folder, so this line is useless (?)

Reply:
Will be fixed in a next PR when it's merged